projects/js-packages/components/components/gridicon/index.jsx

```diff
@@ -161,6 +162,12 @@ class Gridicon extends Component {
 						<path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z" />
 					</g>
 				);
+			case 'gridicons-info-outline':
```

Comment:
Would we need a title for a11y? https://github.com/Automattic/jetpack/pull/24328/files#diff-0436a2487b243e8bfaa997e34e5b15bf441908671083cc83fa214929f1a7180fR40

Reply:
Added in [ed87560](https://github.com/Automattic/jetpack/pull/24328/commits/ed87560991233d6b8347d5396e88d7110e2c31c3) 👍 And for `info` as well as `info-outline.

Reply:
<img width="687" alt="Screen Shot 2022-05-11 at 16 37 45" src="https://user-images.githubusercontent.com/17325/167770234-bd4b64ba-b0db-4400-b10c-79ed7b99bea2.png">